src/Symfony/Component/ExpressionLanguage/ParserCache/ParserCacheInterface.php

```diff
@@ -15,6 +15,8 @@
 
 /**
  * @author Adrien Brault <adrien.brault@gmail.com>
+ *
+ * @deprecated This interface is deprecated. Use instead Psr\Cache\CacheItemPoolInterface PSR6 interface
```

Comment:
` * @deprecated since version 3.2, to be removed in 4.0. Use instead Psr\Cache\CacheItemPoolInterface instead.`